helm_deploy/prisoner-content-hub-backend/templates/db-refresh.yaml

```diff
@@ -0,0 +1,54 @@
+{{- if .Values.dbRefresh.enabled -}}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: db-refresh-script
+  labels:
+    {{- include "prisoner-content-hub-backend.labels" . | nindent 4 }}
+data:
+  entrypoint.sh: |-
+{{ .Files.Get "db-refresh.sh" | indent 4 }}
+---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ include "prisoner-content-hub-backend.fullname" . }}-db-refresh-cronjob
+  labels:
+    {{- include "prisoner-content-hub-backend.labels" . | nindent 4 }}
+spec:
+  schedule: "30 4 * * *"
```

Comment:
Note this is scheduled to run 30 mins after the db backup cronjob. https://github.com/ministryofjustice/prisoner-content-hub-backend/pull/438/files#diff-404543f25cde5fb7ecafaa0a26e02153e3356cdfe443b158a6d038a80e5e0093R20